src/CommunityToolkit/Xamarin.CommunityToolkit/Views/MediaElement/Tizen/MediaPlayerImpl.tizen.cs

```diff
@@ -0,0 +1,438 @@
+		}
+
+		public bool IsMuted
+		{
+			get => player.Muted;
+			set => player.Muted = value;
+		}
+
+		public bool IsLooping
+		{
+			get => player.IsLooping;
+			set => player.IsLooping = value;
+		}
+
+		public int Duration => player.StreamInfo.GetDuration();
+
+		public int Position
+		{
+			get
+			{
+				if (player.State == PlayerState.Idle || player.State == PlayerState.Preparing)
+					return 0;
+				return player.GetPlayPosition();
+			}
+		}
+
+		public DisplayAspectMode AspectMode
+		{
+			get
+			{
+				return aspectMode;
+			}
+
+			set
+			{
+				aspectMode = value;
+				ApplyAspectMode();
+			}
+		}
+
+		bool HasSource => source != null;
+
+		IVideoOutput? VideoOutput
+		{
+			get
+			{
+				return videoOutput;
+			}
+
+			set
+			{
+				if (TargetView != null)
+					TargetView.PropertyChanged -= OnTargetViewPropertyChanged;
+
+				videoOutput = value;
+
+				if (TargetView != null)
+				{
+					TargetView.PropertyChanged += OnTargetViewPropertyChanged;
+				}
+			}
+		}
+
+		VisualElement? TargetView => VideoOutput?.MediaView;
+
+		Task TaskPrepare
+		{
+			get => taskPrepare ?? Task.CompletedTask;
+			set => taskPrepare = value;
+		}
+
+		public event EventHandler? UpdateStreamInfo;
+
+		public event EventHandler? PlaybackCompleted;
+
+		public event EventHandler? PlaybackStarted;
+
+		public event EventHandler<BufferingProgressUpdatedEventArgs>? BufferingProgressUpdated;
+
+		public event EventHandler? PlaybackStopped;
+
+		public event EventHandler? PlaybackPaused;
+
+		public event EventHandler? ErrorOccurred;
+
+		public async Task<bool> Start()
+		{
+			cancelToStart = false;
+			if (!HasSource)
+				return false;
+
+			if (player.State == PlayerState.Idle)
+			{
+				await Prepare();
+			}
+
+			if (cancelToStart)
+				return false;
+
+			try
+			{
+				Device.BeginInvokeOnMainThread(() =>
+				{
+					player.Start();
+					PlaybackStarted?.Invoke(this, EventArgs.Empty);
+				});
+			}
+			catch (Exception e)
+			{
+				Log.Error($"Error On Start : {e.Message}");
+				return false;
+			}
+			return true;
+		}
+
+		public void Pause()
+		{
+			try
+			{
+				player.Pause();
+				PlaybackPaused?.Invoke(this, EventArgs.Empty);
+			}
+			catch (Exception e)
+			{
+				Log.Error($"Error on Pause : {e.Message}");
+			}
+		}
+
+		public void Stop()
+		{
+			cancelToStart = true;
+			_ = ChangeToIdleState();
+			PlaybackStopped?.Invoke(this, EventArgs.Empty);
+		}
+
+		public void SetDisplay(IVideoOutput? output)
+		{
+			VideoOutput = output;
+		}
+
+		public async Task<int> Seek(int ms)
+		{
+			try
+			{
+				await player.SetPlayPositionAsync(ms, true);
+			}
+			catch (Exception e)
+			{
+				Log.Error($"Fail to seek : {e.Message}");
+			}
+			return Position;
+		}
+
+		public void SetSource(Core.MediaSource source)
+		{
+			this.source = source;
+			if (HasSource && AutoPlay)
+			{
+				_ = Start();
+			}
+			else if (!HasSource)
+			{
+				Stop();
+			}
+		}
+
+		public async Task<Stream?> GetAlbumArts()
```

Comment:
This is a good candidate to be a `ValueTask`.
What do think to add the `Async` suffix on this method?
```suggestion
		public async ValueTask<Stream?> GetAlbumArts()
```